paimon-format/src/main/java/org/apache/paimon/format/avro/RowDataToAvroConverters.java

```diff
@@ -146,6 +147,21 @@ public Object convert(Schema schema, Object object) {
                             }
                         };
                 break;
+            case TIMESTAMP_WITH_LOCAL_TIME_ZONE:
+                converter =
+                        new RowDataToAvroConverter() {
+                            private static final long serialVersionUID = 1L;
+
+                            @Override
+                            public Object convert(Schema schema, Object object) {
+                                return ((Timestamp) object)
+                                        .toInstant()
+                                        .atZone(ZoneId.systemDefault())
+                                        .toInstant()
+                                        .toEpochMilli();
```

Comment:
I think for `LocalTimestampMillis` and `LocalTimestampMicros`, the returned long value is different. Here should check the precision.